packages/node_modules/@webex/plugin-meetings/src/member/util.js

```diff
@@ -0,0 +1,117 @@
+import {
+  USER,
+  RESOURCE_ROOM,
+  OBSERVE,
+  NONE,
+  WAIT,
+  LEFT,
+  JOINED,
+  IDLE,
+  IN_LOBBY,
+  NOT_IN_MEETING,
+  IN_MEETING,
+  AUDIO_STATUS,
+  VIDEO_STATUS,
+  SEND_RECEIVE,
+  RECEIVE_ONLY
+} from '../constants';
+
+const MemberUtil = {};
+
+MemberUtil.isUser = (participant) => participant && participant.type === USER;
+
+MemberUtil.isGuest = (participant) => participant && participant.guest;
```

Comment:
does eslint complain about this? should be able to do this for the other functions in the other utils files.

Reply:
yeah it doesn't there but does in the other files, I think it has something to do with es6

Reply:
i resolved it a different way in the other files for clarity